<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainTableWidget</class>
 <widget class="QWidget" name="MainTableWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>781</width>
    <height>70</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainTableWidget</string>
  </property>
  <widget class="QPushButton" name="label_2">
   <property name="geometry">
    <rect>
     <x>17</x>
     <y>14</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:url(&quot;:/res/a.png&quot;);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="label">
   <property name="geometry">
    <rect>
     <x>684</x>
     <y>15</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:url(&quot;:/res/speak.png&quot;);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="RollingLabel" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>679</x>
     <y>10</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgba(0,0,0,0.5);
border-radius:17px;
color:#1191d3;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="pushButton">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>10</y>
     <width>111</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgba(0,0,0,0.5);
border-radius:17px;
color:#caa257;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="table_show">
   <property name="geometry">
    <rect>
     <x>52</x>
     <y>10</y>
     <width>501</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgba(0,0,0,0.5);border-radius:17px;color:#caa257;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="table_show_2">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>10</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgba(0,0,0,0.5);
border-radius:17px;
color:#caa257;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="RollingLabel" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>10</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgba(0,0,0,0.5);
border-radius:17px;
color:#1191d3;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="muyu">
   <property name="geometry">
    <rect>
     <x>728</x>
     <y>18</y>
     <width>19</width>
     <height>19</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:url(&quot;:/res/muyu.png&quot;);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="ClickLabel" name="down_button">
   <property name="geometry">
    <rect>
     <x>293</x>
     <y>52</y>
     <width>191</width>
     <height>7</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:white;
border-radius: 5px;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>pushButton_2</zorder>
  <zorder>label</zorder>
  <zorder>pushButton</zorder>
  <zorder>table_show</zorder>
  <zorder>table_show_2</zorder>
  <zorder>label_2</zorder>
  <zorder>pushButton_3</zorder>
  <zorder>muyu</zorder>
  <zorder>down_button</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RollingLabel</class>
   <extends>QLabel</extends>
   <header>rolling_label.h</header>
  </customwidget>
  <customwidget>
   <class>ClickLabel</class>
   <extends>QLabel</extends>
   <header>clicklabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
